packages/bits/demo/src/components/demo/menu/menu-visual-test/menu-visual-test.component.ts

```diff
@@ -40,6 +40,13 @@ export class MenuVisualTestComponent {
             {title: "Link menu item", itemType: "link", url: "#button", disabled: true},
             {title: "Export PDF", itemType: "link", icon: "export-pdf", url: "#button"},
         ]},
+        {
+            header: "section 2 title", itemsSource: [
```

Comment:
Newline preferred here. Also, do you mind updating the format of the other objects as well at lines 11, 19, and 32?

```suggestion
            header: "section 2 title",
            itemsSource: [
```